action.yml

```diff
@@ -14,6 +14,9 @@ inputs:
   tutor_plugins:
     description: "The list of Tutor indexed plugins to install. E.g: 'plugin1 plugin2'"
     required: false
+  inline_tutor_plugins_folder:
+    description: "Optional path to the folder containing the inline Tutor plugins to install."
```

Comment:
Can you include an example here as in `tutor_plugins`? Is the path relative to the app_name? I think it'd be better illustrated with an example.